cmd/influxd/config.go

```diff
@@ -36,11 +37,21 @@ const (
 
 // Config represents the configuration format for the influxd binary.
 type (
+	Collectd struct {
+		Addr string `toml:"address"`
+		Port uint16 `toml:"port"`
+
+		Database string `toml:"database"`
+		Enabled  bool   `toml:"enabled"`
+		TypesDB  string `toml:"typesdb"`
+	}
```

Comment:
@benbjohnson do you want the graphite block pulled out as well?  Do you want the definitions be be closer to where the methods are defined?